plugins/nodes/src/js/data/NodesNetworkClient.ts

```diff
@@ -32,7 +32,7 @@ export function fetchNodesNetwork(): Observable<
 > {
   const [success, error] = partition(
     (response: RequestResponse<NodesNetworkResponse>) => response.code < 300
-  )(request("/net/v1/nodes"));
+  )(request<NodeNetworkResponse[]>("/net/v1/nodes").pipe(share()));
```

Comment:
can we remove the sharing in `NodesResolver` now?

Reply:
If I don't share I get double the shares.